### PR TITLE
Restaurant Page Project Microverse

### PR DESCRIPTION
**Pull request by Alaska01**
A simple restaurant page was built using Vanilla Javascript to generate its entire content.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,3 +1,51 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Google
+
+## 📝 License
+
+This project is [MIT](lic.url) licensed.
```
Comment (MahmoudBakr23) [inline readme.md]: - [ ] Kindly fix the broken MIT link.
Please note that such small mistakes might leave bad impressions on future viewers. 🙏 😄

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @Alaska01 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct
- Descriptive README file
- Live demo link
- Great design
- Great code separation
- Webpack is well-configured

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️

- [ ] For a professional PR, please change the default headline from a 'Feature branch` to a more descriptive headline.

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

## Hi Aye Aye, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Alaska01** [comment]
Merging into Development. Project Approved!!!!!

**Pull request merged**